hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/BucketCache.java

```diff
@@ -244,6 +242,8 @@
   private static final String FILE_VERIFY_ALGORITHM =
     "hbase.bucketcache.persistent.file.integrity.check.algorithm";
   private static final String DEFAULT_FILE_VERIFY_ALGORITHM = "MD5";
+  private static final String STAT_THREAD_ENABLE_KEY = "hbase.bucketcache.stat.enable";
+  private static final boolean STAT_THREAD_ENABLE_DEFAULT = false;
```

Comment:
And better set it to true to keep the old behavior by default?

Reply:
The purpose to make the log alternative is that the log is only used when heavy debug is needed. Turn on is a little bit annoying at other time I think. So that I changed the old behaviour. What do you think?

Reply:
I think in this PR we'd better keep the old behavior, i.e, set it true. We can file another issue to change it to true on master branch, and mark it as incompatible change. On branch-2.x we'd better keep the old behavior.